hands_on/sync_upstream.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+from exercise_utils.git import add_remote
+from exercise_utils.github_cli import (
+    get_github_username,
+    fork_repo,
+    clone_repo,
+    has_repo,
+    delete_repo
+)
+
+__requires_git__ = True
+__requires_github__ = True
+
+
+TARGET_REPO = "git-mastery/samplerepo-finances"
+LOCAL_DIR = "gitmastery-samplerepo-finances"
+
+def _get_full_repo_name(verbose: bool) -> str:
+    username = get_github_username(verbose)
+    return f"{username}/{LOCAL_DIR}"
+
+
+def download(verbose: bool):
+    full_repo_name = _get_full_repo_name(verbose)
```

Comment:
If you already have this, I would rather not having it as a separate function since the rest of the function will have access to this variable

Reply:
I.e. moving the function behavior directly within the download function here